src/doc/trpl/inline-assembly.md

```diff
@@ -100,6 +100,21 @@ fn main() {
 }
 ```
 
+If you would like to use real operands in this position, however,
+you are required to put curly braces `{}` around the register that
+you want, and you are required to put the specific size of the
```

Comment:
I don't believe this is true, e.g. https://github.com/rust-lang/rust/issues/24799#issuecomment-96145023 .